### PR TITLE
[14.0][MIG] account_payment_order_notification: Migration to 14.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 14.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT40457

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
You can squash first 2 commits together.

File: account_payment_order_notification/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+To configure this module, you need to:
+
+#. Go to *Invoicing > Configuration > Payment Modes* and create a payment mode if you wish.
```
Comment (LudLaf) [inline account_payment_order_notification/readme/CONFIGURE.rst]:
```suggestion
#. Go to *Invoicing > Configuration > Payment Modes* and create a payment mode, if you wish.
```

File: account_payment_order_notification/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+#. Go to *Invoicing > Customers > Customers* and creates a new record as follows:
```
Comment (LudLaf) [inline account_payment_order_notification/readme/CONFIGURE.rst]:
```suggestion
#. Go to *Invoicing > Customers > Customers* and create a new record as follows:
```

File: account_payment_order_notification/readme/CONFIGURE.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    * Name: Test customer 1
+    * Email: customer1@test.com
+
+#. Go to *Invoicing > Customers > Customers* and creates a new record as follows:
```
Comment (LudLaf) [inline account_payment_order_notification/readme/CONFIGURE.rst]:
```suggestion
#. Go to *Invoicing > Customers > Customers* and create a new record as follows:
```

File: account_payment_order_notification/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,11 @@
+#. Go to *Invoicing > Customers > Invoices* and creates one or more invoices linked to the payment mode.
```
Comment (LudLaf) [inline account_payment_order_notification/readme/USAGE.rst]:
```suggestion
#. Go to *Invoicing > Customers > Invoices* and create one or more invoices linked to the payment mode.
```

File: account_payment_order_notification/readme/USAGE.rst [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+#. Go to *Invoicing > Customers > Invoices* select the invoices created and execute the action called "Post entries".
+#. Go to *Invoicing > Customers > Invoices* select the invoices created and execute the action called "Add to Payment/Debit Order" and creates a payment order.
```
Comment (LudLaf) [inline account_payment_order_notification/readme/USAGE.rst]:
```suggestion
#. Go to *Invoicing > Customers > Invoices* select the invoices created and execute the action called "Add to Payment/Debit Order" and create a payment order.
```

**Comment by pedrobaeza** [comment]
@LudLaf please do a complete review

**Review by LudLaf — APPROVED** [review]
Functionality works fine.

**Comment by victoralmau** [comment]
Changes done.

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**